README.md

```diff
@@ -22,10 +22,16 @@ $ npx @cypress/chrome-recorder
 
 The CLI will prompt you to enter the path of the directory or file that you would like to modify as well as a path to write the generated Cypress test to.
 
-If you prefer to enter paths via the CLI, you can run:
+If you prefer to enter paths via the CLI, you can run the following to export individual recordings:
```

Comment:
I would add `command` to this. So it reads like:

`If you prefer to enter paths via the CLI, you can run the following command to export individual recordings`